docs/index.asciidoc

```diff
@@ -3,6 +3,7 @@
 
 :version:        6.0.0-alpha1
 :major-version:  6.x
+:docker-image:   docker.elastic.co/kibana/kibana:6.0.0-alpha1
```

Comment:
No reason at all! I thought that I had tried it and failed, but it must have been something else.

Works fine, and I much prefer it. Thanks.